pkg/autoscaler/metrics/stats_scraper_test.go

```diff
@@ -91,6 +96,30 @@ func TestNewServiceScraperWithClientHappyCase(t *testing.T) {
 	}
 }
 
+func checkBaseStat(t *testing.T, got Stat) {
+	if got.PodName != scraperPodName {
+		t.Errorf("stat.PodName=%v, want %v", got.PodName, scraperPodName)
+	}
+	// (3.0 + 5.0 + 3.0) / 3.0 * 3 = 11
+	if got.AverageConcurrentRequests != 11.0 {
+		t.Errorf("stat.AverageConcurrentRequests=%v, want %v",
+			got.AverageConcurrentRequests, 11.0)
+	}
+	// ((5 + 7 + 5) / 3.0) * 3 = 17
+	if got.RequestCount != 17 {
+		t.Errorf("stat.RequestCount=%v, want %v", got.RequestCount, 15)
+	}
+	// (2.0 + 4.0 + 2.0) / 3.0 * 3 = 8
+	if got.AverageProxiedConcurrentRequests != 8.0 {
+		t.Errorf("stat.AverageProxiedConcurrentRequests=%v, want %v",
+			got.AverageProxiedConcurrentRequests, 8.0)
+	}
+	// ((4 + 6 + 4) / 3.0) * 3 = 14
+	if got.ProxiedRequestCount != 14 {
+		t.Errorf("stat.ProxiedCount=%v, want %v", got.ProxiedRequestCount, 12)
```

Comment:
```suggestion
		t.Errorf("stat.ProxiedRequestCount=%v, want %v", got.ProxiedRequestCount, 12)
```

Reply:
actually:
```suggestion
		t.Errorf("stat.ProxiedRequestCount =%v, want %v", got.ProxiedRequestCount, 14)
```

Reply:
Something was wrong with my typing. But it was pretty late

Reply:
As this one :)